src/CommonLib/OutputTypes/Computer.cs

```diff
@@ -35,7 +35,7 @@ public class ComputerStatus
         public string Error { get; set; }
 
         public static string NonWindowsOS => "NonWindowsOS";
-        public static string OldPwd => "PwdLastSetOutOfRange";
```

Comment:
This an expected deletion? Looks like a constant and wanted to make sure.

Reply:
It got renamed to NotActive, since we're not checking more than just PwdLastSet